test/unit/util.js

```diff
@@ -62,14 +74,14 @@ function promiseFactory(inst, yaw, pitch, fov, answerFile, threshold = 2, isQuat
 		}
 
 		// Then
-		compare(answerFile, inst.canvas, (pct, data) => {
+		compare(answerFile, canvas, (diff, data) => {
 			const result = {
-				success: pct < threshold,
-				difference: pct,
+				success: diff < threshold,
+				difference: diff,
 				threshold
 			};
 
-			if (pct < threshold) {
+			if (result.success) {
```

Comment:
Is **rej(result)** sometimes called?

Reply:
https://github.com/naver/egjs-view360/pull/242/files#diff-05373d53f2ea65d465136bf03eebcd12R94

Should tests proceed in order?
If not, how about using **promise.all**?

```js
Promise.all(tests.map(test => promiseFactory(inst, ...test));
```

Reply:
@younkue 
1. yes, it is
2. it should be proceed in order. it cannot be simultanoesly because canvas should not changed while comparing.